### PR TITLE
Add checklist for creating new standards

### PR DESCRIPTION
**Pull request by annevk**
@domenic @foolip did I miss anything? I thought it would be useful to have this somewhere as we now have multiple candidates for new standards so we'll likely have to exercise this at some point.

### CODE REVIEW

File: NEW-STANDARD.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+When adding a new WHATWG standard, these are the things that need to be done:
+
+1. Create a logo.
+1. Identify a shortname that is used for the spec.whatwg.org subdomain and repository (for tooling it is important that these are named identically).
+   1. Ensure the subdomain works.
+   1. Ensure there is repository.
+1. Create a Twitter account.
+1. Add it to https://github.com/whatwg/sg/blob/main/db.json and ensure the relevant details are filled in. (In case there's a new Workstream more updates will be needed.)
+1. Run https://github.com/whatwg/spec-factory.
```
Comment (foolip) [inline NEW-STANDARD.md:9]: Why is this necessary? Can it be used to create a new empty repo in a previous step?

File: NEW-STANDARD.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+1. Run `labels.py`.
+1. Draw the rest of the fucking owl.
```
Comment (foolip) [inline NEW-STANDARD.md:11]: I laughed, but I don't know about this :) If one hasn't seen the meme it will seem bizarre and maybe not very welcoming. (There's a chance people looking to get engaged will find their way to this file via a search engine.)

File: NEW-STANDARD.md (hunk 1 above)
Comment (annevk) [reply NEW-STANDARD.md:9]: It will (should?) fill in some of the boilerplate files.

File: NEW-STANDARD.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+When adding a new WHATWG standard, these are the things that need to be done:
+
+1. Identify a shortname that is used for the spec.whatwg.org subdomain and repository (for tooling it is important that these are named identically).
+   1. Ensure the subdomain works and is configured: https://github.com/whatwg/misc-server.
+   1. Ensure there is repository.
+1. Create a logo and add it to https://github.com/whatwg/whatwg.org/tree/main/resources.whatwg.org.
```
Comment (ricea) [inline NEW-STANDARD.md:6]: Maybe say the logo should be svg and use the right shade of green?

Comment (annevk) [inline NEW-STANDARD.md:6]:
```suggestion
1. Create an SVG logo in the right shade of green (i.e., WHATWG green) and add it to https://github.com/whatwg/whatwg.org/tree/main/resources.whatwg.org.
```

**Comment by GPHemsley** [comment]
> we now have multiple candidates for new standards

For the record, I found:
whatwg/meta#202 
whatwg/sg#140

Are there any others?

File: NEW-STANDARD.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+When adding a new WHATWG standard, these are the things that need to be done:
```
Comment (domenic) [inline NEW-STANDARD.md:1]:
```suggestion
# New Standard Checklist

When adding a new WHATWG standard, these are the things that need to be done:
```

File: NEW-STANDARD.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+1. Identify a shortname that is used for the spec.whatwg.org subdomain and repository (for tooling it is important that these are named identically).
+   1. Ensure the subdomain works and is configured: https://github.com/whatwg/misc-server.
+   1. Ensure there is repository.
```
Comment (domenic) [inline NEW-STANDARD.md:5]:
```suggestion
   1. Ensure there is a repository under the whatwg GitHub organization.
```

File: NEW-STANDARD.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+1. Create an SVG logo in the right shade of green (i.e., WHATWG green) and add it to https://github.com/whatwg/whatwg.org/tree/main/resources.whatwg.org.
+1. Create a Twitter account.
+   1. Use https://github.com/whatwg/whattweetbot-keys to get the relevant keys. (The Steering Group will help with this.)
+1. Add it to https://github.com/whatwg/sg/blob/main/db.json and ensure the relevant details are filled in. (In case there's a new Workstream more updates will be needed.)
```
Comment (domenic) [inline NEW-STANDARD.md:9]:
```suggestion
1. Add the standard to https://github.com/whatwg/sg/blob/main/db.json and ensure the relevant details are filled in. (In case there's a new Workstream more updates will be needed.)
```

File: NEW-STANDARD.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+1. Run https://github.com/whatwg/spec-factory.
```
Comment (domenic) [inline NEW-STANDARD.md:10]:
```suggestion
1. Run https://github.com/whatwg/spec-factory inside a local checkout of the standard's repository to create the appropriate boilerplate files.
```

although maybe this should be done before creating the repository...

File: NEW-STANDARD.md [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+1. Run `labels.py`.
```
Comment (domenic) [inline NEW-STANDARD.md:11]:
```suggestion
1. Run [`labels.py`](./labels.py) with appropriate `--repository` and `--token` parameters to create appropriate labels for the new repository.
```

File: NEW-STANDARD.md [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+1. You should be all set. When in doubt reach out to the Steering Group.
```
Comment (domenic) [inline NEW-STANDARD.md:12]:
```suggestion

You should be all set. When in doubt reach out to the Steering Group.
```

**Review by domenic — APPROVED** [review]
LGTM with nits

File: NEW-STANDARD.md (hunk 7 above)
Comment (annevk) [reply NEW-STANDARD.md:10]: I don't think we support this option currently, but that might be good to add.

Comment (domenic) [reply NEW-STANDARD.md:10]: Oh, I see. I guess leaving as-is then is fine and people will have to read the docs over there to realize that they should run it in the parent directory (?).

Comment (annevk) [reply NEW-STANDARD.md:12]: I guess I should look at this again (prolly tomorrow) and divide steps a bit between Admin/SG and editor. It's mainly meant as a checklist for Admin/SG, but that is not clear.

**Comment by annevk** [comment]
I have clarified this even further.

**Review by domenic — APPROVED** [review]
